src/main/java/com/thecookiezen/presentation/ClusterController.java

```diff
@@ -43,20 +44,41 @@ public String page() {
         return "clusters/all";
     }
 
-    @RequestMapping(value = "clusters", method = RequestMethod.POST)
-    public String saveOrUpdate(@Valid Cluster cluster, BindingResult bindingResult) {
+    @RequestMapping(value = "clusters", method = RequestMethod.DELETE)
```

Comment:
\[Checkstyle\] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2\.